wire-grpc-client/src/jvmMain/kotlin/com/squareup/wire/internal/grpc.kt

```diff
@@ -228,7 +229,7 @@ internal fun Response.grpcResponseToException(suppressed: IOException? = null):
       }
     }
 
-    return GrpcException(GrpcStatus.get(grpcStatusInt), grpcMessage, grpcStatusDetailsBin, url)
+    return GrpcException(GrpcStatus.get(grpcStatusInt), grpcMessage, grpcStatusDetailsBin?.toByteArray(), url)
```

Comment:
do I need to `.use { it.toByteArray() }` ?

Reply:
Nope